### PR TITLE
[PRD-6159] - upgraded the jars commons-compress and commons-io

### PR DESCRIPTION
**Pull request by lsaikrishna89**

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 15s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: pom.xml [hunk 1]
```diff
@@ -171,7 +171,7 @@
     <xstream.version>1.4.20</xstream.version>
     <jackrabbit.version>2.21.19</jackrabbit.version>
     <oak-jackrabbit.version>1.48.0</oak-jackrabbit.version>
-    <commons-compress.version>1.24.0</commons-compress.version>
+    <commons-compress.version>1.26.1</commons-compress.version>
```
Comment (lucboudreau) [inline pom.xml:174]: 1.26.2 is the latest. Consider using this instead.

**Review by lucboudreau — CHANGES_REQUESTED** [review]

**Review by smaring — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (lucboudreau) [reply pom.xml:174]: Additionally, version 1.26.2 is the correct version to use with commons-io 2.16.1.

Screenshot 2024-06-17 102506

**Comment by lucboudreau** [comment]
@smaring - I think you jumped the gun here. See my comment above. The version of commons-compress they used is not ideal for the version of commons-io. The right version of compress should be 1.26.2.

Screenshot 2024-06-17 102719

**Comment by lucboudreau** [comment]
Please review and merge to fix this.

https://github.com/pentaho/maven-parent-poms/pull/553

**Pull request merged**